src/Search/Search.js

```diff
@@ -3,23 +3,24 @@ import shallow from 'zustand/shallow'
 
 import { useAppStore } from '../App/stores'
 import { Card, Flex, Text, Button } from '../Primitives'
-import { initialFilters } from '../filters'
+import { filterables } from '../filters'
 import FilterGroup from './FilterGroup'
 
 const ORDER = ['title', 'type', 'keywords']
 const sortFilters = (a, b) => ORDER.indexOf(a?.name) - ORDER.indexOf(b?.name)
+const filterablesArr = Object.values(filterables)
```

Comment:
Since `fitlerables` are now stored by label, I just need to put them in an array here. I'm not too worried about this code, as it will likely change once we start worrying about making the UI more configurable.